app/controllers/hq/orphans_controller.rb

```diff
@@ -3,4 +3,63 @@ class Hq::OrphansController < HqController
   def show
     @orphan = Orphan.find(params[:id])
   end
+
+  def index
+    @orphans = Orphan.paginate(:page => params[:page])
+  end
+
+  def create
+    @orphan ||= Orphan.new
```

Comment:
I'm curious about the use of `||=` here: when creating a new record, don't we want to be sure that `@orphan` points to a new instance rather than whatever may have been assigned to it before? Granted, given this controller's structure, such an accident is unlikely. But, again with this controller's structure in mind, is there any particular reason to use `||=` in place of simple `=`?